infra/site-cookbooks/akka-linux-tweaks/recipes/default.rb

```diff
@@ -18,3 +18,9 @@
     sysctl net.core.wmem_max=2097152
   EOH
 end
+
+bash "disable swap" do
+  code <<-EOH
+    swapoff --all
+  EOH
+end
```

Comment:
disabling swap added to akka nodes // @patriknw 
(change being rolled out as we speak)